crates/containerd-shim-wasm/src/container/context.rs

```diff
@@ -44,6 +47,24 @@ pub enum Source<'a> {
     Oci(&'a [WasmLayer]),
 }
 
+impl<'a> Source<'a> {
+    pub fn into_bytes(self) -> anyhow::Result<Cow<'a, Vec<u8>>> {
```

Comment:
Why do we need to consume self?
Could this be `as_bytes(&self)` instead?

Reply:
Ah you're right we don't